[33mWARNING [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:137 Problem reading geckodriver versions: error sending request for url (https://raw.githubusercontent.com/SeleniumHQ/selenium/trunk/common/geckodriver/geckodriver-support.json). Using latest geckodriver version
[33mWARNING [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:137 There was an error managing geckodriver (error sending request for url (https://github.com/mozilla/geckodriver/releases/latest)); using driver found in the cache
[33mWARNING [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:137 Error sending stats to Plausible: error sending request for url (https://plausible.io/api/event)